app/scripts/modules/core/src/delivery/executionGroup/execution/Execution.tsx

```diff
@@ -90,6 +90,9 @@ export class Execution extends React.Component<IExecutionProps, IExecutionState>
           if (this.mounted) {
             const dataSource = application.getDataSource(this.props.dataSourceKey);
             executionService.updateExecution(application, updated, dataSource);
+            if (this.props.dataSourceKey === 'executions' && !execution.isActive) {
```

Comment:
I put it in because we're using the `Execution` component in a weird way to show fast property rollouts. It's not really necessary here, as it'll just no-op when the `removeCompletedExecutionsFromRunningData` operation happens, so I'll remove this block.